packages/@react-aria/steplist/src/useStepList.ts

```diff
@@ -10,19 +10,21 @@
  * governing permissions and limitations under the License.
  */
 
+
+import {AriaStepListProps, StepListAria} from '@react-types/steplist';
+import {filterDOMProps} from '@react-aria/utils';
 import {HTMLAttributes} from 'react';
 // @ts-ignore
 import intlMessages from '../intl/*.json';
-import {StepListAria} from '@react-types/steplist';
 import {useMessageFormatter} from '@react-aria/i18n';
 
-// We don't appear to need any props to this?  It is just providing the default aria-label 
-// As all the aria heavy lifting is do in `useStepListItem`
-export function useStepList(/* <T>( props: StepListProps<T>, state: StepListState<T> */): StepListAria {
+export function useStepList<T>(props: AriaStepListProps<T>): StepListAria {
```

Comment:
Just for reference, Tabs and ActionGroup both have keyboard delegates which get passed to `useSelectableCollection` so that arrow key navigation is properly handled for RTL locales (arrow key directions still follow the appropriate direction (https://github.com/adobe/react-spectrum/blob/main/packages/@react-aria/actiongroup/src/useActionGroup.ts#L41-L50)

Reply:
Thanks for the tip, it looks fairly simple to add.  I'll try it soon.